core-processor/src/executor.rs

```diff
@@ -240,26 +210,25 @@ where
 
     // Execute program in backend env.
     let execute = || {
-        let env = Environment::new(
+        Environment::new(
```

Comment:
if it won't bother you, please split it back by assigning to variable. seems much readable

Reply:
Done